src/services/createRouterFind.spec.ts

```diff
@@ -40,10 +40,3 @@ test('when given a url that does not match a route returns undefined', () => {
 
   expect(route).toBeUndefined()
 })
-
-test('when given an external url that does not match a route returns undefined', () => {
-  const find = createRouterFind(routes)
-  const route = find('https://example.com')
-
-  expect(route).toBeUndefined()
-})
```

Comment:
Is this no longer true?

Reply:
correct, because the route matching rules no longer care about host so this url looks like just an empty string, which matches like 4 different routes

Reply:
Well that's a problem then right? If a user runs with they'll get a route that doesn't match the url they passed in.

Reply:
I guess? we have that external check within the router when actually setting the current route, but no such check for `router.resolve` or `router.find`. Should we move/duplicate that check deeper?